crates/symbolicator/src/services/symbolication.rs

```diff
@@ -1265,11 +1372,24 @@ impl SymbolicationActor {
             let modules: Vec<_> = modules.into_iter().map(|(_, module)| module).collect();
 
             metric!(time_raw("symbolication.num_modules") = modules.len() as u64);
+            metric!(
+                time_raw("symbolication.unusable_modules") =
```

Comment:
We really could do with an alias that calls `time_raw` `histogram` instead...

Reply:
Agreed, we should align this with Relay's metrics macros, with the exception of the enums.

Reply:
hmm, maybe, maybe not.  let's not jump into that right now 😄

Reply:
sorry, being more transparent: i still think metrics-rs is the way to go.  creating a simple alias for a function right now i think is reasonable, spending more time on copying relay is likely not without evaluating things a bit more carefully.  It's not like metrics are a major pain point right now.